events/message.js

```diff
@@ -3,9 +3,10 @@ const cotwActions = require('../eventActions/cotwActions');
 const hocActions = require('../eventActions/hocActions');
 const snapshotActions = require('../eventActions/snapshotActions');
 const sleepclubActions = require('../eventActions/sleepclubActions');
-const profanityActions = require('../eventActions/profanityActions');
+// TEMP const profanityActions = require('../eventActions/profanityActions');
```

Comment:
Should temps be pushed?

Reply:
Would it be possible to have ESLint error if a // TEMP comment is still in?

Reply:
`// TEMP` isn't a special comment or anything, so probably not. I added them because the profanity filter errors were spamming me but I intended to remove them. I had a sticky note with "remove comments on profanity features" on it. Good catch, thank you as always ;-;